app/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL="https://testapi.getpaidinbitcoin.com.au"
```

Comment:
This should be something `REACT_APP_GPIB_URL`

Reply:
We can actually get it for https://github.com/dltxio/idem/blob/master/assets/sites.json#L7

Reply:
Done